homeassistant/components/websocket_api/commands.py

```diff
@@ -238,6 +279,19 @@ async def handle_manifest_get(hass, connection, msg):
         connection.send_error(msg["id"], const.ERR_NOT_FOUND, "Integration not found")
 
 
+@decorators.websocket_command({vol.Required("type"): "integration/setup"})
```

Comment:
Let's make the type more specific. What about:
```suggestion
@decorators.websocket_command({vol.Required("type"): "integration/setup_info"})
```